site/app/views/grading/ElectronicGraderView.php

```diff
@@ -109,9 +109,10 @@ public function statusPage(
             $submitted_total = $num_components > 0 ? $total / $num_components : 0;
             $graded_total = $num_components > 0 ? round($graded / $num_components, 2) : 0;
             if ($peer) {
-                $num_components = count($gradeable->getPeerComponents()) * $gradeable->getPeerGradeSet();
+                $num_components = count($gradeable->getPeerComponents());
                 $graded_total = $num_components > 0 ? $graded / $num_components : 0;
                 $submitted_total = $num_components > 0 ? $total / $num_components : 0;
+                $peer = false;
```

Comment:
Setting peer to false here makes it impossible for your other changes to take affect.

Reply:
Corrected!

Reply:
$peer should not be changed here, as above. Currently the peer score is 0% 0/0 because of this change

![image](https://user-images.githubusercontent.com/11095297/85879053-ae4e2600-b7a7-11ea-98d7-88ca9f5e5d83.png)